en/ros/offboard_control.md

```diff
@@ -14,7 +14,7 @@ There are two things you want to setup on the firmware side before starting offb
 
 ### Map an RC switch to offboard mode activation
 
-To do this, load up the parameters in *QGroundControl* and look for the RC_MAP_OFFB_SW parameter to which you can assign the RC channel you want to use to activate offboard mode.
+In _QGroundControl_ you can set the [RC_MAP_OFFB_SW](../advanced_config/parameter_reference.md#RC_MAP_OFFB_SW) parameter to the RC channel that will be used to activate offboard mode.
 It can be useful to map things in such a way that when you fall out of offboard mode you go into position control.
 
 Although this step isn't mandatory since you can activate offboard mode using a MAVLink message.
```

Comment:
@beniaminopozzan I think you're saying here that you don't need to map a switch because you can do so in other ways? I.e. using GCS or MAVLink? 

Depending on your answer I think I'll modify this as a post process.

Reply:
@hamishwillee I was not saying that but you are right, you can enter offboard mode via GCS, via MAVLink command, via ROS 2 and of course via RC.